Chore/update repo (#2)
valory-xyz/dev-template: README.md

# Dev-template

> A template for development with the open-aea and other agent tools and frameworks.

Reviewer: ```suggestion
A template for development with the open-autonomy framework.
```

## Linters:

>       make formatters

Reviewer: add generators
